app/views/layouts/application.html.erb

```diff
@@ -21,5 +21,10 @@
   </div>
 
 
+
+
+
+
+
```

Comment:
Why these newlines had appeared here? It is important to keep version history clean and readable. Such random lines makes it hard to understand what changes had been made to particular file.